configs/backend/backend.yaml

```diff
@@ -6,7 +6,7 @@ limits:
   distribution: 600
   scanner: 150
 coordinator:
-  addr: 'strelka_coordinator_1:6379'
```

Comment:
does this underscore have issues too? should we use dashes?

Reply:
I checked the Strelka backend container logs and it looks like it finds the coordinator:

```
2023-01-24 15:35:52 2023-01-24 23:35:52 - [INFO] root [strelka-backend.main]: using backend configuration /etc/strelka/backend.yaml
2023-01-24 15:35:52 2023-01-24 23:35:52 - [DEBUG] root [strelka-backend.main]: verified coordinator is up
```